app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -108,10 +108,14 @@ ListingService = ($http, $localStorage, $modal, $q, $state) ->
         incomeLevels.push({
           occupancy: occupancy,
           yearly: parseFloat(amiLevel.amount),
-          monthly: parseFloat(amiLevel.amount) / 12.0
+          monthly: Service._roundDownMaxMonthlyIncome(amiLevel.amount)
         })
     return incomeLevels
 
+  Service._roundDownMaxMonthlyIncome = (amount) ->
```

Comment:
did i get this right? when the story said round down to the nearest dollar, I assumed it means make the last dollar 0 basically? see https://www.pivotaltracker.com/story/show/136159045